packages/gateway/src/graphql/config.ts

```diff
@@ -58,6 +61,14 @@ interface IStringIndexSignatureInterface {
 
 type StringIndexed<T> = T & IStringIndexSignatureInterface
 
+export interface Context {
+  dataSources: {
+    locationsAPI: LocationsAPI
+    practitionerRoleAPI: PractitionerRoleAPI
+  }
+  headers: IAuthHeader
+}
+
```

Comment:
Can we perhaps reuse the type defined in context.ts file?

Reply:
Thanks, I think that I forgot to remove later, sorry! Done in https://github.com/opencrvs/opencrvs-core/pull/4706/commits/270763159f7c733bf6296823867ad7560ffc0b0e